### ISSUE TITLE
zsh: command not found: lightnovel-crawler

### ISSUE THREAD

**Issue by SgtBlade** [dipu-bd/lightnovel-crawler#1256]
After installing lightnovel-crawler the command doesn't seem to work in the terminal.

In issue [672](https://github.com/dipu-bd/lightnovel-crawler/issues/672) it was mentioned that this is related to issue [648](https://github.com/dipu-bd/lightnovel-crawler/issues/648) which said _"PyInquirer depends on regex which is not supported on Mac"_ and _replaced with questionary_ after which the issue was closed however the problem doesn't seem to be fixed.

**Comment by dipu-bd** [comment]
Can you send the error detail? Run with `lncrawl -lll` to display debug log.

**Comment by SgtBlade** [comment]
Apologies, forgot about this. 

I couldn't send the error because the lncrawl command did not work. I made my own fix using the following steps:

1) I downloaded the repo and extracted it in a folder where I plan to keep my scripts.

2) After that I created a new alias using:
`nano ~/.zshrc` in Terminal

3) Once GNU nano was opened I created the following alias
`alias lncrawl="python3 /Users/*my name*/Documents/Scripts/lightnovel-crawler-master/lncrawl/__main__.py"`

4) As a last step I refreshed the Terminal commands using:
`source ~/.zshrc`


It's not an optimal fix but it does work now.
